P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -1023,164 +1023,365 @@ overshadows the historical content</quote>.</egXML>
+          in the sentence; certainly they do not fit into any of the categories so
+          far proposed as reasons for italicizing.  Note also that the word
+          <mentioned>Anglicé</mentioned> is not italicized although it is not
+          generally considered an English word.
+        </p>
+        <p>The following sample encoding for the above passage attempts to take
+          into account all the above points:
+          <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#COHTGEG-eg-02">A pretty common case, I believe; in all <emph>vehement</emph>
+            debatings. She says I am <q rend="italic">too witty</q>;
+            <foreign xml:lang="la" rend="roman">Anglicé</foreign>,
+            <gloss rend="italic">too pert</gloss>; I, that she is
+            <q rend="italic"> too wise</q>; that is to say, being likewise
+            put into English, <gloss rend="italic">not so young as she has
+              been</gloss>: in short, she is grown so much into a
+            <hi rend="italic">mother</hi>, that she had forgotten she ever
+            was a <hi rend="italic">daughter</hi>.</egXML>
+          <!-- WWP proofer points out inconsistency between tagging     -->
+          <!-- and commentary.  "too witty" and "too wise" retagged     -->
+          <!-- from TERM (which the prose says they are *not*) to Q     -->
+          <!-- (which the prose says they *are*).  WWP proofer also     -->
+          <!-- suggests EMPH for 'mother' and 'daughter', but our       -->
+          <!-- imaginary interpretation says we don't understand the    -->
+          <!-- italics there, so we leave it undisambiguated.  (msm)    -->
+        </p></div>
+    </div>
+    
+    
+    <div type="div3" xml:id="COHTGRB">
+        <head>Ruby Annotations</head>
+        
+        <p>The word <mentioned>ruby</mentioned> (or <mentioned>rubi</mentioned>) refers
+          to a particular method of glossing runs of text which is common in East Asian scripts.
+          In horizontally-oriented text, ruby annotations typically appear above the text being 
+          glossed, while in vertical runs of text they may appear to the left or right, or both, 
+          also oriented vertically.  An English example of a ruby annotation might look like this:
+          
+        </p>
+        <figure><graphic width="10rem" url="Images/rbTEI.png" mimeType="image/png"/></figure>
+        
+        <p>In Japanese, furigana (振り仮名) ruby annotations are often used to provide pronunciation 
+          guidance for readers; characters from the largely phonetic hiragana or katakana syllabaries
+          accompany Chinese characters, like this:
+        </p>
+        
+        <figure><graphic width="22rem" url="Images/rbNhkEasy.png" mimeType="image/png"/>
+          <head>The first line of a news story from NHK News Web Easy intended for Japanese
+            learners, in which every Chinese character has a ruby gloss.</head>
```

Comment:
It's not actually true that each character has a gloss; you've found a different problem. In this particular case, it's each WORD consisting of Chinese characters that has a ruby gloss. The ruby is attached to the word rather than the characters in this case. I'll update accordingly.